### DISCUSSION TITLE
isShown, isEnabled no longer supported in v8?

### DISCUSSION THREAD

**Discussion by lstillwe** [TerriaJS/terriajs#6279] [Q&A] [answered]
With TerriaMap version7, on startup, I could automatically load a layer from the data catalog into the workbench and show it on the map using the item attributes - **isShown** and **isEnabled** (set to true). In TerriaMap version 8, these attributes don't seem to be supported.
Does anyone know how can I achieve the same functionality with version 8?

Thanks very much,
Lisa

**Comment by hardreddata** [comment]
Take a look at 

```
"show": true,
"isOpenInWorkbench": true,
```

Though I am unsure about `isEnabled`.

**Reply by hardreddata** [reply]
Actually maybe `show` is doing both...

**Comment by zoran995** [comment, marked as answer]
For loading data by default V8 uses `workbench` array property on the level of initialization files. It contains array of item ids that are shown on workbench by default.
Check https://docs.terria.io/guide/customizing/initialization-files/ and search for `workbench`.
Along with that you can use `show` boolean property on catalog item level, that controls whether the item is visible on the map. Each item should support this property.
i.e. something like this (writing from top of my head), should show dataset in workbench on initial load and it is visible on map (show)
```
{
    "catalog": [
        "type": "wms",
        "id": "dataset-1",
        "show": true
    ],
    "workbench": ["dataset-1"]
}
```
